fastplotlib/layouts/_gridplot.py

```diff
@@ -343,7 +362,25 @@ def show(
                 0, 0
             ].camera.maintain_aspect
 
-        return VBox([self.canvas, self.toolbar.widget])
+        if not sidecar:
+            return VBox([self.canvas, self.toolbar.widget])
+
+        # used when plot.show() is being called again but sidecar has been closed via "x" button
+        # need to force new sidecar instance
+        # couldn't figure out how to get access to "close" button in order to add observe method on click
```

Comment:
how do we know if the user has closed the sidecar with the **x** button?

Reply:
this would indicate that the **x** but was used because you know that the plot instance is still open but `plot.show()` has been called again which would imply that the sidecar has been closed with the **x** or with `plot.sidecar.close()` and not with `plot.close()`

if the sidecar is still open and you are calling `plot.show()` again (which I don't know why you would) but this would open the plot in a new tab of the existing sidecar as to above where I showed that multiple calls will put the plots side-by-side in the same sidecar which looks weird

Reply:
> if the sidecar is still open and you are calling `plot.show()` again (which I don't know why you would) but this would open the plot in a new tab of the existing sidecar as to above where I showed that multiple calls will put the plots side-by-side in the same sidecar which looks weird

Users will always try bizarre things :smile: . I say we go with it, and see what the quirks are when they arise.